### DISCUSSION TITLE
Do we really need RSA encryption for sessions?

### DISCUSSION THREAD

**Discussion by VShawn** [1Remote/1Remote#317] [General]
Yeas ago when I start the APP, I made RSA encryption to keep password safe in sqlite. 

This can ensure that even if the database is been stolen, user data will not be read.

---

When I take a review of this RSA encryption not, I find it once the database is been stolen, OS is no longer safe either.In this situation there is a high probability that the RSA key file will be stolen at the same time. From this point, RSA encryption is pointless.

On the other hand, if user has bitlocker enabled, RSA encryption would also be meaningless.

---

### then can I just make a simple encryption or encoder? 

maybe base64 is ok

THE purpose of this simple encryption is just to avoid storing password in plaintext.

A higher level of security should be guaranteed by the operating system

**Comment by majkinetor** [comment]
I agree with you here. I would just add that safety should not just on the OS level but now with the team access too.

Hence I will close this.
